flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/serializer/AvroToProtoSerializer.java

```diff
@@ -16,12 +16,36 @@
 
 package com.google.cloud.flink.bigquery.sink.serializer;
 
+import com.google.cloud.flink.bigquery.common.utils.SchemaTransform;
 import com.google.cloud.flink.bigquery.sink.exceptions.BigQuerySerializationException;
 import com.google.protobuf.ByteString;
+import org.apache.avro.Schema;
 import org.apache.avro.generic.GenericRecord;
 
 /** Serializer for converting Avro's {@link GenericRecord} to BigQuery proto. */
 public class AvroToProtoSerializer implements BigQueryProtoSerializer<GenericRecord> {
+    /**
+     * Function to convert TableSchema to Avro Schema.
+     *
+     * @param tableSchema A {@link com.google.api.services.bigquery.model.TableSchema} object to
+     *     cast to {@link Schema}
+     * @return Converted Avro Schema
+     */
+    private Schema getAvroSchema(com.google.api.services.bigquery.model.TableSchema tableSchema) {
```

Comment:
Please import `TableSchema` with other imports instead of using fully qualified class path.

Reply:
this file would use TableSchema (`com.google.cloud.bigquery.storage.v1.TableSchema`) as well. Which is why the current Table Schema has been referenced in this way. We could move the upcoming `com.google.cloud.bigquery.storage.v1.TableSchema` usages to a separate class if full class path is not preferred..

Reply:
Ack, fixed.